st2common/st2common/service_setup.py

```diff
@@ -147,6 +147,9 @@ def setup(
 
     LOG.info("Using logging config: %s", logging_config_path)
 
+    LOG.info("Using coordination url: %s", cfg.CONF.coordination.url)
```

Comment:
Would we need to obfuscate the url if there is a password in it?

Reply:
@cognifloyd Good catch, indeed - will add tests for it and obfuscate it.

Reply:
d9e6c5517